test/python/test_wrapper.py

```diff
@@ -32,6 +32,16 @@ def test_wrapper_register_ok(self, QE_TOKEN, QE_URL, hub, group, project):
         self.log.info(backends)
         self.assertTrue(len(backends) > 0)
 
+    def test_offline(self):
+        import string
+        import random
+        FAKE_TOKEN = 'this_token_is_not_going_to_be_sent_anywhere'
+        FAKE_URL = 'http://{0}.com'.format(
```

Comment:
We might be able to remove this test? Generating the random URL always felt a bit weird anyway, and in practice it seems to be nearly the same as the current `test_register_bad_credentials()`: can you take a look of both and see if it makes sense unifying them into one?

Reply:
Thanks. Indeed they are the same test.